Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,49 @@
 import React from 'react';
-import './App.css';
+import './App.scss';
+import ActionButton from './components/ButtonComponents/ActionButton';
+import NumberButton from './components/ButtonComponents/NumberButton';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
 
 const App = () => {
-  return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
-  );
+	let numberButtons = [9, 8, 7, 6, 5, 4, 3, 2, 1];
+	let numButtonStyle = 'num-button';
```

Comment:
Just a small suggestion, that obviously doesn't change a whole lot, but try to use `const` in a place like this. Essentially, use const until it doesn't make sense to. Not super important though.